fitbenchmarking/fitting/misc.py

```diff
@@ -7,17 +7,23 @@
 from fitbenchmarking.utils import fitbm_result
 
 
-def compute_chisq(actual, fitted):
+def compute_chisq(actual, fitted, errors):
     """
     Simple function that calculates the sum of the differences squared
     between the data and the fit.
 
     @param actual :: The values from the known data
     @param fitted :: The values from the fitted data
+    @param fitted :: The values of the errors (weights)
 
     @returns :: the sum of the square of each element in differences
     """
-    chi_sq = np.sum(np.square(fitted - actual))
+    r = fitted - actual
+    if errors:
```

Comment:
I'm not sure this works. Try:
```suggestion
    if errors is not None:
```